akka-http-core/src/test/scala/akka/http/impl/engine/client/pool/PoolGatewaySpec.scala

```diff
@@ -49,6 +49,9 @@ class PoolGatewaySpec extends AkkaSpec {
       val firstFuture = poolGateway.shutdown()
       val secondFuture = poolGateway.shutdown()
 
+      gatewayActorProbe.expectMsgType[Shutdown]
+      gatewayActorProbe.expectNoMessage()
```

Comment:
Good verifying that. Could perhaps trim the time to wait a bit (default is 3 seconds I think) to not make the test slow, 500 ms should be enough.

Reply:
OK, will do